build/files.c

```diff
@@ -2244,6 +2245,12 @@ int readManifest(rpmSpec spec, const char *path, const char *descr, int flags,
 	    continue;
 	if (specExpand(spec, lineno, buf, &expanded))
 	    goto exit;
+	if (flags & STRIP_TRAILINGSPACE) {
+	    char *end = expanded + strlen(expanded) - 1;
+	    while (end >= expanded && isspace((unsigned char)*end))
```

Comment:
Use risspace() instead, it'll also save you from having to include ctype.h

Reply:
Done.